packages/components/src/angle-picker-control/styles/angle-picker-control-styles.js

```diff
@@ -33,17 +34,17 @@ export const CircleIndicatorWrapper = styled.div`
 `;
 
 export const CircleIndicator = styled.div`
-	background: ${ COLORS.ui.border };
+	background: #3858e9;
```

Comment:
Can this reference a variable instead?

Reply:
Unfortunately no, we don't have a variable for this color.